### PR TITLE
Bump all reasources to use kubernetes 0.17.3 ADDENDUM

### PR DESCRIPTION
**Pull request by amuraru**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?

Sync helm charts CRDs definitions following the upgrade to k8s 0.17 libs and controller-gen upgrade

### CODE REVIEW

File: charts/kafka-operator/templates/operator-kafka-crd.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  labels:
-    app.kubernetes.io/name: {{ include "kafka-operator.name" . }}
```
Comment (stoader) [inline charts/kafka-operator/templates/operator-kafka-crd.yaml:5]: These helm specific labels shouldn't be removed. Unfortunately the gen tool used by kubebuilder is not helm aware thus we need to maintain these labels manually to make the helm charts to be proper

Comment (amuraru) [reply charts/kafka-operator/templates/operator-kafka-crd.yaml:5]: Do you feel strong for keeping helm labels in crd @stoader?
The reason I am asking is that we also have an internal patch to move to helm3 where crds are not managed as templates anymore. 

https://geeksocket.in/posts/helm-2-3-migration/ goes at lenght on explaining this. Basically to keep compat between helm2 amd helm3 helm is recommending to move out crds in a separate dir in the chart

Comment (amuraru) [reply charts/kafka-operator/templates/operator-kafka-crd.yaml:5]: I am talking specifically about this change https://github.com/adobe/kafka-operator/commit/9537fce209934f9fed9076a3d36c22288a40a303 we wanna propose

Comment (stoader) [reply charts/kafka-operator/templates/operator-kafka-crd.yaml:5]: Thinking it more I think you're right.

File: charts/kafka-operator/templates/operator-kafka-topic-crd.yaml [hunk 2]
```diff
@@ -1,39 +1,36 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    certmanager.k8s.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "kafka-operator.fullname" . }}-server-cert
```
Comment (stoader) [inline charts/kafka-operator/templates/operator-kafka-topic-crd.yaml:5]: I believe this is needed by cert-manager.

**Pull request merged**